### PR TITLE
Directory restructuring of components in settings, minor UI fixes

### PR DESCRIPTION
**Pull request by Saranya-jena**
Signed-off-by: Saranya-jena <saranya.jena@mayadata.io>

## Proposed changes
- moved `UserDetails` and `ChooseAvatarModal` components to `Components` Directory
- Fixed UI bugs in Settings
- Added the project switching implementation to `View Project` button
Summarize your changes here to communicate with the maintainers and make sure to put the link of that issue

## Types of changes

What types of changes does your code introduce to Litmus? Put an `x` in the boxes that apply
- [ ] New feature (non-breaking change which adds functionality)
- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update (if none of the other choices applies)

## Checklist

Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code.
- [ ] I have read the [CONTRIBUTING](https://github.com/litmuschaos/litmus/blob/master/CONTRIBUTING.md) doc
- [ ] I have signed the commit for DCO to be passed.
- [ ] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works (if appropriate)
- [ ] I have added necessary documentation (if appropriate)

## Dependency
- Please add the links to the dependent PR need to be merged before this (if any).

### CODE REVIEW

**Review by vanshBhatia-A4k9 — DISMISSED** [review]

File: litmus-portal/frontend/src/views/Settings/TeamingTab/Invitation/AcceptedInvitations/index.tsx [hunk 1]
```diff
@@ -48,45 +56,24 @@ const AcceptedInvitations: React.FC = () => {
...
 
   useEffect(() => {
-    const otherProject: Project[] = [];
+    const otherProject: OtherProjectsType[] = [];
     projects.map((project) => {
       return project.members.forEach((member: Member) => {
         if (
           member.user_id === userID &&
           member.role !== 'Owner' &&
           member.invitation === 'Accepted'
```
Comment (SarthakJain26) [inline litmus-portal/frontend/src/views/Settings/TeamingTab/Invitation/AcceptedInvitations/index.tsx:65]: string can be replaced with constant

File: litmus-portal/frontend/src/views/Settings/AccountsTab/AccountSettings/index.tsx [hunk 2]
```diff
@@ -117,200 +117,186 @@ const AccountSettings: React.FC = () => {
...
+          {/* New Password */}
+          <div style={{ marginTop: '1rem' }} />
+          <div data-cy="newPassword">
+            <InputField
+              required
+              type="password"
+              onChange={handleNewPassword('newPassword')}
+              helperText={
+                validateStartEmptySpacing(password.newPassword)
+                  ? 'Should not start with empty space'
```
Comment (SarthakJain26) [inline litmus-portal/frontend/src/views/Settings/AccountsTab/AccountSettings/index.tsx:152]: Translation required

File: litmus-portal/frontend/src/views/Settings/AccountsTab/AccountSettings/index.tsx [hunk 3]
```diff
@@ -117,200 +117,186 @@ const AccountSettings: React.FC = () => {
...
+          {/* Confirm new password */}
+          <div style={{ marginTop: '1rem' }} />
+          <div data-cy="confPassword">
+            <InputField
+              helperText={
+                validateConfirmPassword(
+                  password.newPassword,
+                  password.confNewPassword
+                )
+                  ? 'Password is not same'
```
Comment (SarthakJain26) [inline litmus-portal/frontend/src/views/Settings/AccountsTab/AccountSettings/index.tsx:177]: Translation required

File: litmus-portal/frontend/src/views/Settings/AccountsTab/AccountSettings/index.tsx [hunk 4]
```diff
@@ -117,200 +117,186 @@ const AccountSettings: React.FC = () => {
...
+              )}
+              variant={
+                validateConfirmPassword(
+                  password.newPassword,
+                  password.confNewPassword
+                )
+                  ? 'error'
+                  : isSuccess.current
+                  ? 'success'
+                  : 'primary'
```
Comment (SarthakJain26) [inline litmus-portal/frontend/src/views/Settings/AccountsTab/AccountSettings/index.tsx:194]: strings can be replaced with constants

File: litmus-portal/frontend/src/views/Settings/AccountsTab/AccountSettings/index.tsx [hunk 5]
```diff
@@ -117,200 +117,186 @@ const AccountSettings: React.FC = () => {
...
+              }
+              label={t(
+                'settings.accountsTab.accountsSettings.label.newPassword'
+              )}
+              variant={
+                validateStartEmptySpacing(password.newPassword)
+                  ? 'error'
+                  : isSuccess.current
+                  ? 'success'
+                  : 'primary'
```
Comment (SarthakJain26) [inline litmus-portal/frontend/src/views/Settings/AccountsTab/AccountSettings/index.tsx:163]: strings can be replaced with constants

**Review by SarthakJain26 — CHANGES_REQUESTED** [review]

File: litmus-portal/frontend/src/views/Settings/TeamingTab/Invitation/AcceptedInvitations/index.tsx (hunk 1 above)
Comment (Saranya-jena) [reply litmus-portal/frontend/src/views/Settings/TeamingTab/Invitation/AcceptedInvitations/index.tsx:65]: I have already made the changes, in an upcoming PR

**Review by SarthakJain26 — APPROVED** [review]

**Review by vanshBhatia-A4k9 — APPROVED** [review]

**Review by ishangupta-ds — APPROVED** [review]

**Comment by Othniel-dell** [comment]
Agree to this pull

**Pull request merged**